### DISCUSSION TITLE
Data Reverted to a Previous Point in Time After Power Outage

### DISCUSSION THREAD

**Discussion by orztrickster** [openzfs/zfs#16884] [Q&A]
I installed TrueNAS Scale on Hyper-V using an eight-disk, 4TB-per-disk raidz3 configuration, with the current data utilization at around 95%. After a power outage on December 6, I discovered that the system’s data had reverted to a previous state. Once power was restored, all data added between October 27 and December 6 had vanished.

Previously, a daily snapshot was scheduled, but now all snapshots and the corresponding data from that period are missing. I have already tried running a zpool scrub, but the data did not return. Has anyone experienced a similar situation, or does anyone have a possible solution?
![nas](https://github.com/user-attachments/assets/3e26af09-9e0b-4240-a010-cb2eb894a582)

**Comment by orztrickster** [comment]
This is what it looks like in the console.
![NAS3](https://github.com/user-attachments/assets/e369bb63-9890-42e2-9049-4977b14b0095)

**Comment by amotin** [comment]
I wonder if snapshots could be deleted by replication of something else.  You might want to check `zpool history` for something interesting.
